apps/web/src/components/Svg/GlassGlobeIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import { Svg, SvgProps } from '@pancakeswap/uikit'
+
+export const GlassGlobeIcon: React.FC<SvgProps> = (props) => {
```

Comment:
this can be moved to its own icon in the uikit package. we have already have an Icons component folder which holds all of the icons used throught the interface. 

you can find all icons at 
https://github.com/pancakeswap/pancake-frontend/tree/develop/packages/uikit/src/components/Svg/Icons

i notice there is another icon in this Svg folder here. that was actually added by me when i was unfamiliar wth the uikit collection. you can remove this from web and add to the uikit. after this pr gets merged i will move mine and delete this foler entirely since as of now it only contains the refreshcon